### PR TITLE
Javascript/javascript1/week4

### PR DESCRIPTION
**Pull request by DlnyaMazhari**

### CODE REVIEW

File: javascript/javascript1/week4/Week4HW.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    const daypattern = { day: "numeric", month: "long", year: "numeric" };
+
+    return today.toLocaleDateString(undefined, daypattern);
+  } else {
+    return "I dont understand what you are saying";
+  }
+}
+
+function arithOperator(command) {
+  var digit = command.match(/\d/g);
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/Week4HW.js]: Use let or const instead of var

File: javascript/javascript1/week4/Week4HW.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
+"use strict";
+
+var arrNames = [];
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/Week4HW.js]: Use let or const instead of var

File: javascript/javascript1/week4/Week4HW.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+var todoArr = [];
+
+function getReplay(command) {
+  const splitedCommand = command.split(" ");
+  const userName = splitedCommand[splitedCommand.length - 1];
+  if (command === "" || typeof command != "string") {
+    return "Please say something valid!";
+  }
+  if (command == `Hello my name is ${userName}`) {
```
Comment (ajgcph2019) [inline javascript/javascript1/week4/Week4HW.js:21]: What you did is correct. There is also an alternative like below 
There are string methods like 'startsWith' and 'endsWith'.
For example 
 const commandWords = inputString.split(' ');

  if (inputString.startsWith('hello my name is')) {
    name = commandWords[commandWords.length - 1];
    return `Nice to meet you ${name}`;
  }

**Review by ajgcph2019 — CHANGES_REQUESTED** [review]
Good job

File: javascript/javascript1/week4/Week4HW.js (hunk 1 above)
Comment (DlnyaMazhari) [reply javascript/javascript1/week4/Week4HW.js]: done.

File: javascript/javascript1/week4/Week4HW.js (hunk 2 above)
Comment (DlnyaMazhari) [reply javascript/javascript1/week4/Week4HW.js]: done

File: javascript/javascript1/week4/Week4HW.js (hunk 3 above)
Comment (DlnyaMazhari) [reply javascript/javascript1/week4/Week4HW.js:21]: thank you , I didn't know about startswith and now I know how to use it. this way has been added as a note as well.

**Pull request merged**